packages/orchestrator-ui-components/src/components/WfoSubscription/WfoSubscriptionDetailTree.tsx

```diff
@@ -154,9 +178,8 @@ export const WfoSubscriptionDetailTree = ({
                         </EuiCallOut>
                     )}
                     {selectedIds.length !== 0 &&
-                        selectedIds.map((id, index) => {
-                            const block = idToNodeMap[selectedIds[index]];
-
+                        selectedIds.sort(sortByTree).map((id, index) => {
```

Comment:
Replace `.sort()` with `.toSorted()` to avoid add behavior

Reply:
In this case that is actual overkill. selectedIds is just an array of numbers.